thrimshim/thrimshim/main.py

```diff
@@ -276,8 +277,9 @@ def get_row(ident):
 		and response["video_title"].startswith(title_header)
 	):
 		response["video_title"] = response["video_title"][len(title_header):]
-	description_playlist_re = re.compile(r"\n\n({}\n(- .* \[https://youtube.com/playlist\?list=[A-Za-z0-9_-]+\]\n)+\n)?{}$".format(
+	description_playlist_re = re.compile(r"\n\n({}\n(- .* \[https://youtube.com/playlist\?list=[A-Za-z0-9_-]+\]\n)+\n)?{}\n{}$".format(
```

Comment:
thumbnail section needs to be optional, and include the actual text.
This assumes thumbnail attributions do not contain newlines.
```suggestion
	description_playlist_re = re.compile(r"\n\n({}\n(- .* \[https://youtube.com/playlist\?list=[A-Za-z0-9_-]+\]\n)+\n)?({}[^\n]+\n)?{}$".format(
```